src/gpgservice.py

```diff
@@ -0,0 +1,69 @@
+import gnupg
+import json
+import pprint
+import os
+
+class GPGService:
+
+	def __init__(self):
+		super(GPGService, self).__init__()
+		self.gpg = gnupg.GPG(gnupghome ='homegpg')
+		with open('keys.json','a+') as dataFile:
+			if os.stat('keys.json').st_size is not 0:
+				self.keysFp = json.load(dataFile)
+			else:
+				self.keysFp = {}
+
+	def createKeys(self, domain):
+		self.inputData = self.gpg.gen_key_input(
+				name_email = domain,
+				passphrase = 'password')
```

Comment:
Is it possible to store some other metadata with the key? Like it's purpose (let's say: key for communication with Alice or key for communication with Bob).